test/classes/ferguson/delete-free/owned-shared-fields2.bad

```diff
@@ -1,4 +1,8 @@
-owned-shared-fields2.chpl:9: error: implicit copy-initialization invoked for a type that does not allow it
-owned-shared-fields2.chpl:18: error: implicit copy-initialization invoked for a type that does not allow it
-owned-shared-fields2.chpl:13: error: implicit copy-initialization invoked for a type that does not allow it
-owned-shared-fields2.chpl:22: error: implicit copy-initialization invoked for a type that does not allow it
+owned-shared-fields2.chpl:11: internal error: CAL0078 chpl version 1.18.0 pre-release (573fcb6282)
```

Comment:
It's unfortunate that this went from a user error to an internal error.  Since it's just a .bad file I think we can let it slide.